model/financing.go

```diff
@@ -0,0 +1,276 @@
+/**
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model
+
+type Locate struct {
+	// The currency symbol
+	Symbol string `json:"symbol"`
+	// The available quantity located
+	Quantity string `json:"quantity"`
+	// The interest rate for located symbol
+	Rate string `json:"rate"`
+}
+
+type MarginCallRecord struct {
+	// The unique ID of the margin call
+	MarginCallId string `json:"margin_call_id"`
+
+	// The initial margin call amount in notional value
+	InitialNotionalAmount string `json:"initial_notional_amount"`
+
+	// The outstanding margin call amount in notional value
+	OutstandingNotionalAmount string `json:"outstanding_notional_amount"`
+
+	// The time the margin call is created in RFC3339 format
+	CreatedAt string `json:"created_at"`
+
+	// The time the margin call is due in RFC3339 format
+	DueAt string `json:"due_at"`
+}
+
+type LoanInfo struct {
+	// The unique ID of the portfolio
+	PortfolioId string `json:"portfolio_id"`
+
+	// The currency symbol
+	Symbol string `json:"symbol"`
+
+	// Balance amount
+	Amount string `json:"amount"`
+
+	// Notional balance amount
+	NotionalAmount string `json:"notional_amount"`
+
+	// Settlement due date
+	DueDate string `json:"due_date"`
+}
+
+type MarginAddOnType string
+
+const (
+	MarginAddOnTypeUnspecified     MarginAddOnType = "MARGIN_ADD_ON_TYPE_UNSPECIFIED"
+	MarginAddOnSingleCoinStress    MarginAddOnType = "SINGLE_COIN_STRESS"
+	MarginAddOnConcentrationStress MarginAddOnType = "CONCENTRATION_STRESS"
+	MarginAddOnMacroStress         MarginAddOnType = "MACRO_STRESS"
+	MarginAddOnShortBiasedStress   MarginAddOnType = "SHORT_BIASED_STRESS"
+)
+
+type LoanType string
+
+const (
+	LoanTypeTypeUnspecified     LoanType = "LOAN_TYPE_UNSET"
+	LoanTypeBilateralLending    LoanType = "BILATERAL_LENDING"
+	LoanTypeTradeFinance        LoanType = "TRADE_FINANCE"
+	LoanTypePortfolioMargin     LoanType = "PORTFOLIO_MARGIN"
+	LoanTypeShortCollateralLoan LoanType = "SHORT_COLLATERAL_LOAN"
+	LoanTypeShortCollateral     LoanType = "SHORT_COLLATERAL"
+)
+
+type PMAssetInfo struct {
```

Comment:
Change to 

PmAssetInfo

Keep omitempty for anything that's optional. If it's in the request and will be in the response, don't set.